src/storage/src/mem_table.rs

```diff
@@ -46,15 +51,19 @@ pub enum KeyOp {
 /// `MemTable` is a buffer for modify operations without encoding
 #[derive(Clone)]
 pub struct MemTable {
-    pub(crate) buffer: BTreeMap<Bytes, KeyOp>,
+    pub(crate) buffer: BTreeMap<TableKey<Bytes>, KeyOp>,
```

Comment:
Do we need `#[derive(EstimateSize)]` for TableKey to make memory statistics more accurate?

Reply:
It is already done in this PR [here](https://github.com/risingwavelabs/risingwave/pull/11943/files#diff-e8546b48ec0e07c8f2c4d526da0e83dc956d44ec12929864691d9b82ed06418aR415).